tasks/eth/ink-002-set-respected-game-type/.env

```
@@ -0,0 +1,4 @@
+ETH_RPC_URL="https://ethereum.publicnode.com"
+#`OWNER_SAFE` represents the address of the `deputyGuardian` gnosis safe attached to the `DeputyGuardianModule`
+OWNER_SAFE=0x9BA6e03D8B90dE867373Db8cF1A58d2F7F006b3A # guardian multisig
+SAFE_NONCE=98
```

Going forward let's only use this format, since the `SAFE_NONCE=` format is not always supported (it is off by default for nested sigs). Please double check the address though!

```suggestion
SAFE_NONCE_0X9BA6E03D8B90DE867373DB8CF1A58D2F7F006B3A=98
```

---

it matches the foundation operations safe and this is the respected game type so it looks good